hotxlfp/formulas/statistical.py

```diff
@@ -86,7 +87,8 @@ def MEDIAN(*args):
 
 @dispatcher.register_for('MIN')
 def MIN(*args):
-    return min(utils.inumbers(args))
+    tensors = [torch.tensor(val, dtype=torch.double) for val in args]
```

Comment:
This is to support the case where values are not provided as tensors, like min(1, 2) instead of min([1], [2]). However, we don't use this case anymore. Do you think we should just drop support for this, and change the existing test cases to be torch tensors?

Reply:
i'd rather we be defensive and accept both until 

1. we have some kind of input sanitization that guarantees that everything we pass to hotxlfp is a tensor
2. we have audited the functions in the library to ensure that all of them return tensors

long term, we probably want some kind of Term object to pass around these functions which contains type information and has a tensor/string/whatever as an attribute